src/openms/source/ANALYSIS/TOPDOWN/FLASHDeconvAlgorithm.cpp

```diff
@@ -1036,9 +1078,9 @@ namespace OpenMS
     return deconvolved_spectrum_;
   }
 
-  void FLASHDeconvAlgorithm::setDummyIndex(PeakGroup::DummyIndex flag, FLASHDeconvAlgorithm& targetFD)
+  void FLASHDeconvAlgorithm::setTargetDummyType(PeakGroup::TargetDummyType target_dummy_type, FLASHDeconvAlgorithm& targetFD)
```

Comment:
why is a reference to an algorithm passed here?

Reply:
fixed to pass just necessary data structure (deconvolvedSpectrum).